util/measures/2019/generated-ecqm-data.json

```diff
@@ -451,7 +483,7 @@
         }
       }
     ],
-    "metricType": "singlePerformanceRate"
```

Comment:
Does this file not need the same strata as the `measures-data.json`? Would adding that keep you from having to do the conditional check and pushing in the `{ name: '18+' }` strata in the`nrich-measures-data.js` code?

Reply:
This file is generated from a separate script that takes in a zip file with measures. The measures themselves do not have a separate `12-17` and `18+` strata. There is another file at `util/measure/2019/quality-strata.csv` with strata names to associate with measure id's. But it associates them so by linking them with a numerator UUID. Since the IG doesn't actually have a numerator UUID for 18+ I manually pushed the name `18+` once it gets to the correct measure. I could see if I could take the name from the file that passes in that strata name if the manual push is a concern.

Reply:
I'm mostly just curious and since that file is 2019 specific I don't see any real problem with just pushing into the array